acceptance/tests/lookup/lookup_rich_values.rb

```diff
@@ -20,6 +18,10 @@
 hierarchy:
   - name: Test
     data_hash: rich_data_test
+  - name: Test2
+    data_hash: some_mod::rich_data_test2
```

Comment:
This entry is somewhat unorthodox. Not sure it should be allowed since it references a function provided by a module.